src/main/java/core/basesyntax/HelloWorld.java

```diff
@@ -1,8 +1,55 @@
 package core.basesyntax;
 
-/**
- * Feel free to remove this class and create your own.
- */
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.operation.BalaceOperationHandler;
+import core.basesyntax.operation.OperationHandler;
+import core.basesyntax.operation.PurchaseOperationService;
+import core.basesyntax.operation.ReturnOperationService;
+import core.basesyntax.operation.SupplyOperationHandler;
+import core.basesyntax.service.CreateFileServise;
+import core.basesyntax.service.ParseService;
+import core.basesyntax.service.ReaderService;
+import core.basesyntax.service.ReportService;
+import core.basesyntax.service.WriterService;
+import core.basesyntax.service.impl.CreateFileServiceImpl;
+import core.basesyntax.service.impl.ParseServiseImpl;
+import core.basesyntax.service.impl.ReaderFileServiceImpl;
+import core.basesyntax.service.impl.ReportServiceImpl;
+import core.basesyntax.service.impl.WriterServiceImpl;
+import core.basesyntax.strategy.OperationStrategy;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
 public class HelloWorld {
+    private static final String FILE_WITH_DAY_ACTIVITIES = "src/main/resources/file.csv";
+    private static final String REPORT_FILE = "src/main/resources/report.csv";
+
+    public static void main(String[] args) {
+        Map<String, OperationHandler> map = new HashMap<>();
+        map.put("b", new BalaceOperationHandler());
```

Comment:
What do you think about constants instead of letters in the map?
It's the same for all letters.

Reply:
I think it doesn't matter and I use letters for one time. That's why I decided didn't use constants. let the meters judge us)

Reply:
I agree with you and wrote my opinion just to think if you can move them to constants.